src/commands/environments/destroy.ts

```diff
@@ -68,7 +68,7 @@ export default class EnvironmentDestroy extends BaseCommand {
     const { args, flags } = await this.parse(EnvironmentDestroy);
 
     const account = await AccountUtils.getAccount(this.app, flags.account);
-    const get_environment_options: GetEnvironmentOptions = { environment_name: flags.environment, strict: flags.strict };
+    const get_environment_options: GetEnvironmentOptions = { environment_name: args.environment, strict: flags.strict };
```

Comment:
Thank you for catching this. This change is necessary because `destroy.ts` does not extend the `EnvironmentUtils.flags`, so `environment` would have to be an arg instead of a flag.